apps/passport-client/components/screens/DevconnectCheckinByIdScreen.tsx

```diff
@@ -0,0 +1,361 @@
+import {
```

Comment:
This file is a copy of `DevconnectCheckin.tsx`, with only the minimal changes necessary to support the ID-based checkin flow rather than checking in using the entire PCD.